src/routes/brands/fetchAllBrands.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from "fastify";
+import BrandModel from "../../models/brand";
+import { Brand } from "../../types/Brand";
+
+type Reply = Brand[] | { error: {} };
+type FetchAllBrands = {
+  Reply: Reply;
+};
```

Comment:
@leandro0day I'd like to propose a potentially better name for this could be something like the suggestion below. Moreover, it appears that the **Reply** type might remain relatively stable, so it could be incorporated as part of the suggested  FetchAllBrandsResponse type.

**Suggestion:** Instead of **FetchAllBrands**, we could consider renaming it to something like FetchAllBrandsResponse for better clarity and consistency.

@Ralexs0096, being more experienced in TypeScript, your input would be greatly appreciated. Do you think the proposed naming change aligns with TypeScript best practices and provides improved readability?

Your insights would be valuable in making this decision. 
```suggestion
type FetchAllBrandsResponse = {
  Reply: Brand[] | { error: {} };
};
```

Reply:
No, this name is not referring to the response. It's the type used by the handler. Response type is taken from `/types` directory (generated from /schemas). I'll add a new file about this project's references and standards.